src/nvim/ex_cmds.h

```diff
@@ -19,13 +21,47 @@
 #define ECMD_LAST       (linenr_T)-1    /* use last position in all files */
 #define ECMD_ONE        (linenr_T)1     /* use first line */
 
+/// for cmdl_progress in incsubstitute
+typedef enum {
+  ICS_NO_WD,                 /// state of cmdline when none words are typed
```

Comment:
Capitals here. Also documentation that is put after what it is documenting is marked with `///<`, not `///`. `///<` should be used on all lines.

And style guide suggests using `kICSPatternStart`, `kICSPattern`, `kICSReplacementStart`, `kICSReplacement`. Current names are incorrect from the style guide point of view *and* confusing: “words” you are referring to have good names “pattern” and “replacement string”, no need to use indexes. And no need to fear too long constant names.